portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +28,22 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// Causes the bubble to slide down the page.
+function bubbleFall() {
+    // Find the bubble.
+    const bubbleElem = document.getElementById('bubble');
+    let height = 0;
+    // Make the movement visible and fluid by moving every 10 milliseconds;
+    const id = setInterval(lowerBubbleOnePixel, 10);
+    // For each movement, move the element one pixel down. At the bottom, stop moving.
+    function lowerBubbleOnePixel() {
+        if (height == 1000){
```

Comment:
Oops, sorry I missed this earlier. Should have a space after ) and before {. If you could fix this in another commit/PR that'd be great? 

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

Reply:
Of course! Thanks for catching it.